socorro/unittest/processor/test_processor_app.py

```diff
@@ -277,6 +277,48 @@ def mocked_save_raw_and_processed(*_):
         assert captured_exception.__class__ == ValueError
         assert captured_exception.args[0] == 'Someone is wrong on the Internet'
 
+    @mock.patch('socorro.lib.raven_client.raven')
+    def test_transform_get_error_with_raven_configured_successful(self, mock_raven):
```

Comment:
This test mirrors the other one, except that it covers fetching crashes and the other one covers saving crashes.